src/simulation.cpp

```diff
@@ -322,10 +326,12 @@ Particle*  device_particles {nullptr};
 
 void allocate_banks()
 {
-  if (settings::run_mode == RunMode::EIGENVALUE) {
+  if (settings::run_mode == RunMode::EIGENVALUE || settings::run_mode == RunMode::FIXED_SOURCE) {
```

Comment:
These are the only two run modes that will get you here, so I think you can just remove the `if` statement.